lintly/parsers.py

```diff
@@ -412,6 +412,64 @@ def parse_violations(self, output):
         return violations
 
 
+class TerrascanParser(BaseLintParser):
+    """
+    Terrascan JSON format
+    {
+        "results": {
+            "violations": [
+                {
+                     "rule_name": "apiGatewayName",
+                     "description": "Enable AWS CloudWatch Logs for APIs",
+                     "rule_id": "AWS.API Gateway.Logging.Medium.0567",
+                     "severity": "MEDIUM",
+                     "category": "Logging",
+                     "resource_name": "this",
+                     "resource_type": "aws_api_gateway_stage",
+                     "file": "api_gateway_config.tf",
+                     "line": 15
+                }
+            ],
+            "skipped_violations": null,
+            "scan_summary": {
+                "file/folder": "/Users/vnandusekar/git/terraform-examples/aws/aws_lambda_api",
```

Comment:
I don't know how you feel about this, but I'd leave personal things (like usernames) out and edit to something else

Reply:
Yes you are right actually. I am fixing this